### ISSUE TITLE
Ajaxify grouped product option

### ISSUE THREAD

**Issue by sftsk** [woothemes/woocommerce#7751]
In `class-wc-meta-box-product-data.php` there is a huge performance issue when there are a lot of products with a lot of variations ~line 538:
```
$grouped_products = get_posts( $args );
```
In our store with ~3000 grouped products with ~22000 simple products this line takes ~20 seconds pure php processing time on a local developing machine (MacBook Pro Intel Core i7 2,4 GHz).
It's not much better on a decent production server.

Would it be possible to load this through ajax/ not create a post for each one of the products to get this working faster?

Also this gets loaded even for grouped products which can't even be connected to another product which I guess is a bug and should be fixed as well.

Thank you for your help and for this great plugin!

**Comment by sftsk** [comment]
using something like this already speeds up by a few seconds but is still not great:

```
foreach ( $posts_in as $product ) {
  if ( $product == $post->ID ) {
    continue;
  }
  $post_parents[$product] = get_the_title($product);
}
```

**Comment by mikejolley** [comment]
This could be ajaxified like other product inputs.

**Comment by BFTrick** [comment]
This could definitely be useful but it isn't critical to 2.4. I say we punt it.

**Comment by claudiosmweb** [comment]
@mikejolley @BFTrick This is very simple and can help in performance, so just fix it to 2.4.
